pkgs/by-name/mi/miru/update.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq gnused
+
+set -euo pipefail
+
+ROOT="$(dirname "$(readlink -f "$0")")"
+if [[ ! "$(basename $ROOT)" == "miru" || ! -f "$ROOT/package.nix" ]]; then
+    echo "ERROR: Not in the miru folder"
+    exit 1
+fi
+
+PACKAGE_NIX="$ROOT/package.nix"
+LINUX_NIX="$ROOT/linux.nix"
+DARWIN_NIX="$ROOT/darwin.nix"
+
+MIRU_LATEST_VER="$(curl -s "https://api.github.com/repos/ThaUnknown/miru/releases/latest" | jq -r .tag_name | sed 's/^v//')"
+MIRU_CURRENT_VER="$(grep -oP 'version = "\K[^"]+' "$PACKAGE_NIX")"
+
+if [[ "$MIRU_LATEST_VER" == "null" ]]; then
+    echo "error: could not fetch miru latest version from GitHub API"
```

Comment:
The `update.py` script only displays the content of stderr when an error occurs.

```suggestion
    echo "error: could not fetch miru latest version from GitHub API" >&2
```

Reply:
Good to know, thanks!

Reply:
@d4ilyrun fixed